lib/libraries_map.rb

```diff
@@ -0,0 +1,20 @@
+# frozen_string_literal: true
+
+# Map library codes to labels exported from Folio
+class LibrariesMap
+  def self.for(key)
+    return if key == 'SUL' # do not index building_facet on electronic resources
+
+    key = 'LANE' if key == 'LANE-MED' # If symphony codes are passed as input, we need to update to the folio code
+
+    library = Folio::Types.libraries.values.find { |candidate| candidate.fetch('code') == key }
+    return library.fetch('name') if library
+
+    Honeybadger.notify('Lookup for library not found', context: { key: })
```

Comment:
```suggestion
```
This seems like to swamp honeybadger if we're wrong, and fixing it is outside of our control (and ought to happen when we receive updated library information from FOLIO, if things haven't gone incredibly wrong)

Reply:
@cbeer  Can you explain "ought to happen"? I'm having trouble picturing the situation where we don't have this data.  That seems like something worth documenting.

Reply:
If someone adds a new library code and starts using it, but it isn't in that cached list of libraries (that we only updated daily)...?